lib/rules/decorator-position.js

```diff
@@ -1,20 +1,23 @@
 'use strict';
 
-const { CLIEngine } = require('eslint');
+const { CLIEngine: v7AndEarlier, ESLint: v8Plus } = require('eslint');
 
 const cli = new CLIEngine();
+const ESLint = v7AndEarlier || v8Plus;
```

Comment:
I got rid of the postinstall script and did this.

postinstall script won't run on folks' machines who have ignore-scripts enabled